fiftyone/utils/moge.py

```
@@ -0,0 +1,383 @@
+            mask = masks[i] if masks is not None and i < len(masks) else None
+            if mask is not None:
+                mask = np.asarray(mask) > 0
+
+            if not np.all(np.isfinite(depth)):
+                depth = np.where(np.isfinite(depth), depth, 0)
+
+            if mask is not None and mask.shape == depth.shape:
+                depth = np.where(mask, depth, 0)
+
+            if size is not None and depth.shape[:2] != (height, width):
+                depth = _resize(depth, size, nearest=False)
```

Ruff (0.16.3)

[warning] `for` loop variable `depth` overwritten by assignment target

(PLW2901)